packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash-dashboard/sqm-tax-and-cash-dashboard.feature

```diff
@@ -0,0 +1,109 @@
+      | false        | United States    | n/a              | Not registered. Only participants representing a company in countries that enforce indirect tax (e.g. GST, HST, VAT) must add their indirect tax information. |                        | N/A               |
+      | false        | Papua New Guinea | n/a              | Not registered. Only participants representing a company in countries that enforce indirect tax (e.g. GST, HST, VAT) must add their indirect tax information. |                        | N/A               |
+
+  @minutia
+  Scenario Outline: Participant is registered for indirect tax in Spain
+    When the participant is registered for indirect tax in Spain
+    Then the <registeredDetails> display with <subRegion>
+    And they <doHaveIncomeTaxNumber> from step 2
+    Then the indirect tax section displays <indirectTaxNumbers>
+
+    Examples: 
+      | registeredDetails    | subRegion      | doHaveIncomeTaxNumber | indirectTaxNumbers            |
+      | Registered in Spain, | Spain Proper   | true                  | VAT Number, Income Tax Number |
+      | Registered in Spain, | Canary Islands | false                 | VAT Number, N/A               |
+
+  @minutia @ui
+  Scenario Outline: A Danger Alert is displayed if the users tax form is invalid
+    Given the document has status <status>
+    Then a danger alert indicating the <documentType> with a <taxAlertHeader> and <taxAlertMessage> appears
+
+    Examples: 
+      | status     | documentType | taxAlertHeader                                                             | taxAlertMessage                                                                                                                 |
+      | NOT_ACTIVE | W9           | Your W9 tax form has personal information that doesn't match your profile. | Please resubmit a new W9 form.                                                                                                  |
+      | NOT_ACTIVE | W8-BEN       | W8-BEN tax form is invalid.                                                | Your tax form may have expired or has personal information that doesn’t match your profile. Please submit a new W-8 BEN form.   |
+      | NOT_ACTIVE | W8-BEN-E     | W8-BEN tax form is invalid.                                                | Your tax form may have expired or has personal information that doesn’t match your profile. Please submit a new W-8 BEN-E form. |
+
+  @minutia @ui
+  Scenario: Invoices table is available for participants regsistered for Indirect Tax
+    Given a participant is registered for Indirect Tax
+    Then the Invoices section appears with a table
+    But if they are not registered
+    Then the Invoice section does not appear.
+
```

Comment:
Missing specs for who can go back and edit their payment information and their whole flow